packages/time-utilities/src/lib/Cron.ts

```diff
@@ -0,0 +1,138 @@
+/* eslint-disable @typescript-eslint/restrict-plus-operands */
+import { Time, partRegex, wildcardRegex, allowedNum, predefined, cronTokens, tokensRegex } from './constants';
+
+/**
+ * Handles Cron strings and generates dates based on the cron string provided.
+ * @see https://en.wikipedia.org/wiki/Cron
+ */
+export class Cron {
+	public cron: string;
+	public normalized: string;
+	public minutes: number[];
+	public hours: number[];
+	public days: number[];
+	public months: number[];
+	public dows: number[];
```

Comment:
not related to review, whats `dows`?

Reply:
If I had to guess... Days of Week? I'm guessing this was in diri code too. No idea for sure tho